build-tools/packages/build-cli/src/commands/release/setPackageTypesField.ts

```diff
@@ -171,9 +167,6 @@ function updatePackageJsonTypes(
 			}
 
 			if (filePath) {
-				if (checkFileExists && !fs.existsSync(filePath)) {
-					throw new Error(`${filePath} path does not exists`);
-				}
```

Comment:
Isn't this checking if the types file actually exists - the output file, not the entrypoint file (usually index.d.ts) that api-extractor uses. I think that's a useful check, even if in CI we want to skip it.

Reply:
`ignoreMissingEntryPoint` checks if file exists: https://api.rushstack.io/pages/api-extractor.iextractorconfigprepareoptions.ignoremissingentrypoint/